OloEngine/tests/scripts/perf_trend.py

```
@@ -0,0 +1,130 @@
+
+
+REPO_ROOT = pathlib.Path(__file__).resolve().parents[3]
+HISTORY_DIR = REPO_ROOT / "OloEngine" / "tests" / "Rendering" / "PropertyTests" / "perf_history"
+
+
+def percentile(values, q):
+    """Simple nearest-rank percentile for short lists."""
+    if not values:
+        return float("nan")
+    s = sorted(values)
+    k = max(0, min(len(s) - 1, int(round((q / 100.0) * (len(s) - 1)))))
```

Ruff (0.15.10)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)